main.js

```diff
@@ -18,4 +19,285 @@ function close() {
+    liveLink: 'https://supreeti.github.io/PortfolioMobileFirst/',
+  },
+
+  {
+    id: 1,
+    name: ['Multi-Post Stories', 'Multi-Post Stories'],
+    description: [
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. A daily selection of privately personalized reads; no accounts or sign-ups required.', 'A daily selection of privately personalized reads; no accounts or sign-ups required. A daily selection of privately personalized reads; no accounts or sign-ups required. A daily selection of privately personalized reads; no accounts or sign-ups required.'],
+    image: ['images/Snapshoot Portfolio (1).png', 'images/Snapshoot PortfolioDesk2.png'],
+    technologies: ['html', ' css ', 'javaScript'],
+    technologiesDesk: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    technologiesPop: ['html', ' css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    Source: 'https://github.com/supreeti/PortfolioMobileFirst',
+    liveLink: 'https://supreeti.github.io/PortfolioMobileFirst/',
+  },
+
+  {
+    id: 2,
+    name: ['Tonic', 'Facebook 360'],
+    description:
+      ['A daily selection of privately personalized reads; no accounts or sign-ups required. A daily selection of privately personalized reads; no accounts or sign-ups required.', "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR... Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR..."],
+    image: ['images/Snapshoot Portfolio (2).png', 'images/Snapshoot Portfoliodesk3.png'],
+    technologies: ['html', ' css ', 'javaScript'],
+    technologiesDesk: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    technologiesPop: ['html', ' css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    Source: 'https://github.com/supreeti/PortfolioMobileFirst',
+    liveLink: 'https://supreeti.github.io/PortfolioMobileFirst/',
+  },
+
+  {
+    id: 3,
+    name: ['Multi-Post Stories', 'Uber Navigation'],
+    description:
+      ['A daily selection of privately personalized reads; no accounts or sign-ups required. A daily selection of privately personalized reads; no accounts or sign-ups required.', 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car. A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.'],
+    image: ['images/Snapshoot Portfolio (3).png', 'images/Snapshoot Portfoliodesk4.png'],
+    technologies: ['html', ' css ', 'javaScript'],
+    technologiesDesk: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    technologiesPop: ['html', ' css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    Source: 'https://github.com/supreeti/PortfolioMobileFirst',
+    liveLink: 'https://supreeti.github.io/PortfolioMobileFirst/',
+  },
+];
```

Comment:
- [ ] Kindly make sure you fix the broken image for the below card to make your application look more professional
![Screenshot from 2023-02-10 14-27-23](https://user-images.githubusercontent.com/83042742/218092769-bdf98244-3483-415c-b19e-5ca916504318.png)